osu.Game/Screens/Play/Player.cs

```diff
@@ -155,7 +164,38 @@ private void load(AudioManager audio, OsuConfigManager config, OsuGame osu)
                     },
                     Children = new Drawable[]
                     {
-                        new SkipButton(firstObjectTime) { AudioClock = decoupledClock },
+                        letterboxInBreaksContainer = new Container

```

Comment:
Please move this implementation to a new container; you're polluting `Player` again.